### ISSUE TITLE
pcsaft_ares units

### ISSUE THREAD

**Issue by cassguff** [zmeri/PC-SAFT#113]
Hello,

In doing some tests comparing to literature values, it seems like pcsaft_ares() returns values more consistent with ares/(RT) which is unitless, but the documentation says that it returns J/mol. Which is correct? And if it is actually ares/RT and just a typo in the documentation, does that mean the other residual properties are also divided by RT?

**Comment by zmeri** [comment]
No, the units are J/mol. When developing the code I tested values for energy parameters against values I got from PC-SAFT implemented in Aspen Plus and compared in units of J/mol.
